README.md

```diff
@@ -413,7 +413,32 @@ const client = createClient({
 </details>
 
 <details id="apollo-client">
-<summary><a href="#apollo-client">🔗</a> Client usage with <a href="https://www.apollographql.com">Apollo</a></summary>
+<summary><a href="#apollo-client">🔗</a> Client usage with <a href="https://www.apollographql.com/docs/react/">Apollo Client Web</a> v3.5.10+</summary>
+
+```typescript
+// Direct support for graphql-ws added in Apollo Client Web v3.5.10.
+// See the next section for older versions.
+import { GraphQLWsLink } from "@apollo/client/link/subscriptions";
+import { createClient } from 'graphql-ws';
+
+const link = new WebSocketLink(createClient({
```

Comment:
Should this be `GraphQLWsLink`? Don't see it used anywhere.

Reply:
Good catch, fixed.

Reply:
ps: say hi to Marla for me :)

Reply:
@glasser I work closely with her and will do!